### ISSUE TITLE
RPC reply id quoted twice?

### ISSUE THREAD

**Issue by shesek** [ElementsProject/lightning#2166]
@renepickhardt reported seeing this:

```
$ lightning-cli funds
lightning-cli: Malformed response '{"jsonrpc": "2.0", "error": "Error while processing funds", "id": ""lightning-cli-9289""}
```

It looks like something is incorrectly quoting the reply id with two `""`?

**Comment by renepickhardt** [comment]
I was about to file a report however I was not able to trace that bug down. It seems like the ID field is set in plugin.c but it might be related to the currently fast changing API as I am also running a nodded version

**Comment by cdecker** [comment]
Should be fixed by #2159

**Comment by cdecker** [comment]
Seems to be fixed, please let me know should this problem still persist :wink:
